server/server.js

```diff
@@ -21,35 +22,50 @@ app.use(bodyParser.urlencoded({
 app.use('/static', express.static(path.join(__dirname, '/static')));
 
 
+
+var bobList = [];
+
+
+function nowPlusXMinutes(numMinutes) {
```

Comment:
maybe incrementMinutes(numMinutes){
} 
or put this function inside the bob model? (but i don't know why you made this function so yea)